### PR TITLE
Remove dead line

### LINKED ISSUE #124: Demo removes <?php line
Live demo: https://getrector.org/demo/97209fdb-db60-4425-9278-ca2c00e7836c#result

PHP snippet:
```php
<?php declare(strict_types=1);

class CommonDemoFile
{
	public function run(array $params)
    {

    }
}

class DemoFile extends CommonDemoFile
{
    public function run(array $params)
    {
        return $params;
    }
}
```

```yml
# rector.yaml
parameters:
    sets:
        - 'dead-code'
```

Unexpected: Rector removes `<?php declare(strict_types=1);`

### PR DESCRIPTION
**Pull request by TomasVotruba**
Closes #124

### CODE REVIEW

File: src/FormDataFactory/DemoFormDataFactory.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+final class DemoFormDataFactory
+{
+    public function createFromRectorRun(?RectorRun $rectorRun): DemoFormData
+    {
+        if ($rectorRun) {
+            return new DemoFormData($rectorRun->getContent(), $rectorRun->getConfig());
+        }
+
+        // default values
+        $demoContent = FileSystem::read(__DIR__ . '/../../data/DemoFile.php');
```
Comment (JanMikes) [inline src/FormDataFactory/DemoFormDataFactory.php]: Ultimately these paths could be passed as parameters via constructor, so it is easier to test this class.

File: tests/FormDataFactory/DemoFormDataFactoryTest.php [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+final class DemoFormDataFactoryTest extends AbstractKernelTestCase
+{
+    /**
+     * @var DemoFormDataFactory
+     */
+    private $demoFormDataFactory;
+
+    protected function setUp(): void
+    {
+        self::bootKernel(GetRectorKernel::class);
```
Comment (JanMikes) [inline tests/FormDataFactory/DemoFormDataFactoryTest.php]: `$this->bootKernel()` should be used instead, `bootKernel()` method is not static. PHPStorm is reporting this to me as bug.

Comment (JanMikes) [reply tests/FormDataFactory/DemoFormDataFactoryTest.php]: I see that symfony's `KernelTestCase` is static but in symplify `AbstractKernelTestCase` it is not static

File: src/FormDataFactory/DemoFormDataFactory.php (hunk 1 above)
Comment (TomasVotruba) [reply src/FormDataFactory/DemoFormDataFactory.php]: Parameters suggest it's configurable, which is not.
Also `FileSystem` is 3rd party code, we don't need to test it.

Might be constants though to make it more clear :+1: Thanks for the hint

File: tests/FormDataFactory/DemoFormDataFactoryTest.php (hunk 2 above)
Comment (TomasVotruba) [reply tests/FormDataFactory/DemoFormDataFactoryTest.php]: Yes indeed :+1: Thanks

File: src/FormDataFactory/DemoFormDataFactory.php (hunk 1 above)
Comment (TomasVotruba) [reply src/FormDataFactory/DemoFormDataFactory.php]: Also, parameters or compositive paths (`$someDir . '/path.php') suck at file jumping:

jump

File: public/assets/prism/prism.js [hunk 3]
```diff
@@ -5,7 +5,7 @@ Prism.languages.markup={comment:/<!--[\s\S]*?-->/,prolog:/<\?[\s\S]+?\?>/,doctyp
...
-Prism.languages.diff={coord:[/^(?:\*{3}|-{3}|\+{3}).*$/m,/^@@.*@@$/m,/^\d+.*$/m],deleted:/^[-<].*$/m,inserted:/^[+>].*$/m,diff:{pattern:/^!(?!!).+$/m,alias:"important"}};
+Prism.languages.diff={coord:[/^(?:\*{3}|-{3}|\+{3}).*$/m,/^@@.*@@$/m,/^\d+.*$/m],deleted:/^[-].*$/m,inserted:/^[+>].*$/m,diff:{pattern:/^!(?!!).+$/m,alias:"important"}};
```
Comment (TomasVotruba) [inline public/assets/prism/prism.js]: This took me 30 mins to discover, terrible :D

```diff
- remove
-?php remove

< remove
<?php remove
```

**Pull request merged**